app/assets/stylesheets/app/map.css.scss

```diff
@@ -13,3 +13,13 @@
   background-color: rgba(#fff, 0.8) !important;
   margin-bottom: -80px; // to account for lost vertical space in document flow
 }
+
+#directory_map {
```

Comment:
Avoid using id selectors<br>Selector `directory_map` should be written in lowercase with hyphens